apps/cookbook/src/app/examples/grid-layout-example/grid-layout-multiple-containers-example/grid-layout-multiple-containers-example.component.html

```diff
@@ -0,0 +1,70 @@
+<main>
+  <h1>Example Content 3</h1>
```

Comment:
Consider changing to the following for clarity, or add another heading above it
```suggestion
  <h1>Grid Layout - multiple grid containers</h1>
```